### ISSUE TITLE
0.5.2 glith with Hardware AA off

### ISSUE THREAD

**Issue by MiCkSoftware** [googlesamples/cardboard-unity#96]
Hi!

Here is what I see (Moto x2 and N5) when I set Hardware AA off in quality settings .

Seems linked to 
```C#
rt.antiAliasing = Mathf.Max(QualitySettings.antiAliasing, 1);
```

BTW, I propose to set mip maps of for this render texture:
```C#
rt.useMipMap = false;
```

This happens with both Unity and Native post process

![screenshot_2015-10-16-11-12-09](https://cloud.githubusercontent.com/assets/12721372/10538324/1582379c-73f8-11e5-898d-2596ddf4c058.png)

**Comment by smdol** [comment]
Any useful lines in adb logcat?  If you comment the antialiasing line, is the problem resolved?

**Comment by MiCkSoftware** [comment]
Commenting resolves nothing. I'll check adb later on.

**Comment by MiCkSoftware** [comment]
And what's your pov regarding setting mip maps off? I think I gained 1 FPS.

Cheers!

**Comment by smdol** [comment]
That seems reasonable since we don't render the texture to a mesh that is flat in Z, and there is no anisotropy.  I just thought that mipMaps are off by default, because I tried turning them on once and my frame rate took a huge dive.

**Comment by smdol** [comment]
If you commented out the line "rt.antiAliasing = ..." and it still rendered like in your screenshot, I'm not sure how we know that the problem is with that line?  It might be the RT format is not supported on that phone.  (That's one thing you might see in the adb logcat.)

**Comment by MiCkSoftware** [comment]
Yes. More tests shows that the trigger is only ```QualitySettings.antiAliasing```
When set to 0 render texture has glitches on left camera.

OK for mipmaps.

**Comment by MiCkSoftware** [comment]
OK! A SettingsEventHandler lead me to the wrong conclusion.In the end the problem turns to be related to **Direct render**. The glitches are there when it is disabled.

Also I noticed (see the log) that rendertexture is bigger than screen resolution. Is it intended to be so ?

adb log:

```
--------- beginning of system

--------- beginning of main

D/Unity   ( 7901): ASensorManager_destroyEventQueue returned 0

E/Unity   ( 7901): RenderTexture warning: Destroying active render texture. Switching to main context.

E/Unity   ( 7901):  

E/Unity   ( 7901): (Filename: ./Runtime/GfxDevice/opengles/FrameBufferGLES.cpp Line: 509)

E/Unity   ( 7901): 

D/Unity   ( 7901): [EGL] Attaching window :0x0

I/Unity   ( 7901): onDestroy

I/Unity   ( 8046): splash_mode = 0 (integer)

I/Unity   ( 8046): hide_status_bar = True (bool)

I/Unity   ( 8046): useObb = False (bool)

D/Unity   ( 8046): surfaces: attach glview

I/Unity   ( 8046): Choreographer available: Enabling VSYNC timing

I/Unity   ( 8046): onResume

D/Unity   ( 8046): [EGL] Attaching window :0xb8713ba8

D/Unity   ( 8046): [EGL] Attaching window :0xb8713ba8

I/Unity   ( 8046): windowFocusChanged: true

D/Unity   ( 8046): SystemInfo CPU = ARMv7 VFPv3 NEON, Cores = 4, Memory = 1866mb

D/Unity   ( 8046): SystemInfo ARM big.LITTLE configuration: 4 big (mask: 15), 0 little (mask: 0)

D/Unity   ( 8046): [SHA1] a9285076e8e6959cc2d6014e50c02ec7c30b6e3d   21437243 bytes ~ lib/armeabi-v7a/libunity.so

D/Unity   ( 8046): [SHA1] be1e8966a70792614511acb30b0202138fb4381c      44013 bytes ~ lib/armeabi-v7a/libmain.so

D/Unity   ( 8046): [SHA1] 2e180ffaf29e4247f8b1cf1e9e19d6125ffb900b    3754400 bytes ~ lib/armeabi-v7a/libmono.so

D/Unity   ( 8046): [SHA1] ab15a21e2bb2234d101e53c76871d556ce9f47dc   25161569 bytes ~ lib/x86/libunity.so

D/Unity   ( 8046): [SHA1] 18c51796daaee662122b6235e02529ecc98bea64      27052 bytes ~ lib/x86/libmain.so

D/Unity   ( 8046): [SHA1] 55e8f57d0898595384d30a4c8b48582d86ea9f13    3636444 bytes ~ lib/x86/libmono.so

D/Unity   ( 8046): Mono path[0] = '/data/app/com.mick.we.trailerb-2/base.apk/assets/bin/Data/Managed'

D/Unity   ( 8046): Mono config path = 'assets/bin/Data/Managed'

D/Unity   ( 8046): PlayerConnection initialized from /data/app/com.mick.we.trailerb-2/base.apk/assets/bin/Data (debug = 0)

D/Unity   ( 8046): PlayerConnection initialized network socket : 0.0.0.0 55383

D/Unity   ( 8046): PlayerConnection initialized unix socket : Unity-com.mick.we.trailerb

D/Unity   ( 8046): Multi-casting "[IP] 192.168.0.21 [Port] 55383 [Flags] 2 [Guid] 515888026 [EditorId] 1701664046 [Version] 1048832 [Id] AndroidPlayer(motorola_XT1092@192.168.0.21) [Debug] 0" to [225.0.0.222:54997]...

D/Unity   ( 8046): InitializeScriptEngine OK (afd9bee0)

D/Unity   ( 8046): PlayerConnection already initialized - listening to [192.168.0.21:55383]

D/Unity   ( 8046): PlayerInitEngineNoGraphics OK

D/Unity   ( 8046): GfxDevice: creating device client; threaded=1

D/Unity   ( 8046): [EGL] Request: ES 2.0 RGBA32 8888 0/0

D/Unity   ( 8046): [EGL] Found: ID[5] ES 2.0 RGBA32 8888 0/0 AFBT

D/Unity   ( 8046): ANativeWindow: (1920/1080) RequestedResolution: (0/0) EGLSurface: (1920/1080)

D/Unity   ( 8046): Renderer: Adreno (TM) 330

D/Unity   ( 8046): Vendor:   Qualcomm

D/Unity   ( 8046): Version:  OpenGL ES 3.0 V@84.0 AU@05.00.02.042.016 (CL@)

D/Unity   ( 8046): GLES:     3

D/Unity   ( 8046): GL_AMD_compressed_ATC_texture GL_AMD_performance_monitor GL_AMD_program_binary_Z400 GL_EXT_debug_label GL_EXT_debug_marker GL_EXT_discard_framebuffer GL_EXT_robustness GL_EXT_texture_format_BGRA8888 GL_EXT_texture_type_2_10_10_10_REV GL_NV_fence GL_OES_compressed_ETC1_RGB8_texture GL_OES_depth_texture GL_OES_depth24 GL_OES_EGL_image GL_OES_EGL_sync GL_OES_EGL_image_external GL_OES_element_index_uint GL_OES_fbo_render_mipmap GL_OES_fragment_precision_high GL_OES_get_program_binary GL_OES_packed_depth_stencil GL_OES_depth_texture_cube_map GL_OES_rgb8_rgba8 GL_OES_standard_derivatives GL_OES_texture_3D GL_OES_texture_float GL_OES_texture_half_float GL_OES_texture_half_float_linear GL_OES_texture_npot GL_OES_vertex_half_float GL_OES_vertex_type_10_10_10_2 GL_OES_vertex_array_object GL_QCOM_alpha_test GL_QCOM_binning_control GL_QCOM_driver_control GL_QCOM_perfmon_global_mode GL_QCOM_extended_get GL_QCOM_extended_get2 GL_QCOM_tiled_rendering GL_QCOM_writeonly_rendering GL_EXT_sRGB GL_EXT_sRGB_write_control GL_EXT_

D/Unity   ( 8046): texture_sRGB_decode GL_EXT_texture_filter_anisotropic GL_EXT_multisampled_render_to_texture GL_EXT_color_buffer_float GL_EXT_color_buffer_half_float GL_EXT_disjoint_timer_query 

D/Unity   ( 8046): ETC2 1

D/Unity   ( 8046): OPENGL LOG: Creating OpenGL ES 2.0 graphics device

D/Unity   ( 8046): InitializeGfxDevice OK

D/Unity   ( 8046): Initialize engine version: 5.2.1f1 (44735ea161b3)

D/Unity   ( 8046): AudioManager: Selecting AudioTrack output...

D/Unity   ( 8046): Begin MonoManager ReloadAssembly

D/Unity   ( 8046): Platform assembly: /data/app/com.mick.we.trailerb-2/base.apk/assets/bin/Data/Managed/UnityEngine.dll (this message is harmless)

D/Unity   ( 8046): Loading /data/app/com.mick.we.trailerb-2/base.apk/assets/bin/Data/Managed/UnityEngine.dll into Unity Child Domain

D/Unity   ( 8046): Platform assembly: /data/app/com.mick.we.trailerb-2/base.apk/assets/bin/Data/Managed/Assembly-CSharp-firstpass.dll (this message is harmless)

D/Unity   ( 8046): Loading /data/app/com.mick.we.trailerb-2/base.apk/assets/bin/Data/Managed/Assembly-CSharp-firstpass.dll into Unity Child Domain

D/Unity   ( 8046): Platform assembly: /data/app/com.mick.we.trailerb-2/base.apk/assets/bin/Data/Managed/Assembly-CSharp.dll (this message is harmless)

D/Unity   ( 8046): Loading /data/app/com.mick.we.trailerb-2/base.apk/assets/bin/Data/Managed/Assembly-CSharp.dll into Unity Child Domain

D/Unity   ( 8046): Platform assembly: /data/app/com.mick.we.trailerb-2/base.apk/assets/bin/Data/Managed/Assembly-UnityScript-firstpass.dll (this message is harmless)

D/Unity   ( 8046): Loading /data/app/com.mick.we.trailerb-2/base.apk/assets/bin/Data/Managed/Assembly-UnityScript-firstpass.dll into Unity Child Domain

D/Unity   ( 8046): Platform assembly: /data/app/com.mick.we.trailerb-2/base.apk/assets/bin/Data/Managed/Assembly-UnityScript.dll (this message is harmless)

D/Unity   ( 8046): Loading /data/app/com.mick.we.trailerb-2/base.apk/assets/bin/Data/Managed/Assembly-UnityScript.dll into Unity Child Domain

D/Unity   ( 8046): Platform assembly: /data/app/com.mick.we.trailerb-2/base.apk/assets/bin/Data/Managed/UnityEngine.UI.dll (this message is harmless)

D/Unity   ( 8046): Loading /data/app/com.mick.we.trailerb-2/base.apk/assets/bin/Data/Managed/UnityEngine.UI.dll into Unity Child Domain

D/Unity   ( 8046): Platform assembly: /data/app/com.mick.we.trailerb-2/base.apk/assets/bin/Data/Managed/UnityEngine.Networking.dll (this message is harmless)

D/Unity   ( 8046): Loading /data/app/com.mick.we.trailerb-2/base.apk/assets/bin/Data/Managed/UnityEngine.Networking.dll into Unity Child Domain

D/Unity   ( 8046): Platform assembly: /data/app/com.mick.we.trailerb-2/base.apk/assets/bin/Data/Managed/System.Core.dll (this message is harmless)

D/Unity   ( 8046): - Completed reload, in  0.154 seconds

D/Unity   ( 8046): PlayerInitEngineGraphics OK

D/Unity   ( 8046): Found 23 native sensors

D/Unity   ( 8046): Sensor :        Accelerometer ( 1) ; 0.004790 / 0.01s ; MPU6515 3-axis Accelerometer / InvenSense 

D/Unity   ( 8046): Sensor :        Accelerometer ( 1) ; 0.004790 / 0.01s ; MPU6515 3-axis Accelerometer / InvenSense 

D/Unity   ( 8046): Requested framebuffer: resolution[1920x1080], rgba[8/8/8/8], depth+stencil[on], samples[1]

D/Unity   ( 8046): Created framebuffer: resolution[1920x1080], rgba[8/8/8/8], depth+stencil[24/8], samples[0] 

D/Unity   ( 8046): Platform assembly: /data/app/com.mick.we.trailerb-2/base.apk/assets/bin/Data/Managed/System.dll (this message is harmless)

I/Unity   ( 8046): Awakening level trailer-boot

I/Unity   ( 8046): UnityEngine.Debug:Internal_Log(Int32, String, Object)

I/Unity   ( 8046): UnityEngine.Debug:Log(Object)

I/Unity   ( 8046): GameSettings:Awake() (at C:\Users\TO102081\Workspaces\unity\weons-prologue\Assets\Whispering Eons\Scripts\GameSettings.cs:38)

I/Unity   ( 8046):  

I/Unity   ( 8046): (Filename: C Line: 0)

I/Unity   ( 8046): 

D/Unity   ( 8046): Sensor :        Accelerometer ( 1) ; 0.004790 / 0.01s ; MPU6515 3-axis Accelerometer / InvenSense 

I/Unity   ( 8046): Starting level trailer-boot

I/Unity   ( 8046): UnityEngine.Debug:Internal_Log(Int32, String, Object)

I/Unity   ( 8046): UnityEngine.Debug:Log(Object)

I/Unity   ( 8046): GameSettings:Start() (at C:\Users\TO102081\Workspaces\unity\weons-prologue\Assets\Whispering Eons\Scripts\GameSettings.cs:65)

I/Unity   ( 8046):  

I/Unity   ( 8046): (Filename: C Line: 0)

I/Unity   ( 8046): 

D/Unity   ( 8046): Setting up 1 worker threads for Enlighten.

D/Unity   ( 8046):   Thread -> id: b8b35818 -> priority: 1 

I/Unity   ( 8046): windowFocusChanged: false

I/Unity   ( 8046): onPause

D/Unity   ( 8046): ASensorManager_destroyEventQueue returned 0

E/Unity   ( 8046): RenderTexture warning: Destroying active render texture. Switching to main context.

E/Unity   ( 8046):  

E/Unity   ( 8046): (Filename: ./Runtime/GfxDevice/opengles/FrameBufferGLES.cpp Line: 509)

E/Unity   ( 8046): 

D/Unity   ( 8046): [EGL] Attaching window :0x0

I/Unity   ( 8046): onDestroy

I/Unity   ( 8333): splash_mode = 0 (integer)

I/Unity   ( 8333): hide_status_bar = True (bool)

I/Unity   ( 8333): useObb = False (bool)

D/Unity   ( 8333): surfaces: attach glview

I/Unity   ( 8333): Choreographer available: Enabling VSYNC timing

I/Unity   ( 8333): onResume

D/Unity   ( 8333): [EGL] Attaching window :0xb8713d88

D/Unity   ( 8333): [EGL] Attaching window :0xb8713d88

I/Unity   ( 8333): windowFocusChanged: true

D/Unity   ( 8333): SystemInfo CPU = ARMv7 VFPv3 NEON, Cores = 4, Memory = 1866mb

D/Unity   ( 8333): SystemInfo ARM big.LITTLE configuration: 4 big (mask: 15), 0 little (mask: 0)

D/Unity   ( 8333): [SHA1] a9285076e8e6959cc2d6014e50c02ec7c30b6e3d   21437243 bytes ~ lib/armeabi-v7a/libunity.so

D/Unity   ( 8333): [SHA1] be1e8966a70792614511acb30b0202138fb4381c      44013 bytes ~ lib/armeabi-v7a/libmain.so

D/Unity   ( 8333): [SHA1] 2e180ffaf29e4247f8b1cf1e9e19d6125ffb900b    3754400 bytes ~ lib/armeabi-v7a/libmono.so

D/Unity   ( 8333): [SHA1] ab15a21e2bb2234d101e53c76871d556ce9f47dc   25161569 bytes ~ lib/x86/libunity.so

D/Unity   ( 8333): [SHA1] 18c51796daaee662122b6235e02529ecc98bea64      27052 bytes ~ lib/x86/libmain.so

D/Unity   ( 8333): [SHA1] 55e8f57d0898595384d30a4c8b48582d86ea9f13    3636444 bytes ~ lib/x86/libmono.so

D/Unity   ( 8333): Mono path[0] = '/data/app/com.mick.we.trailerb-2/base.apk/assets/bin/Data/Managed'

D/Unity   ( 8333): Mono config path = 'assets/bin/Data/Managed'

D/Unity   ( 8333): PlayerConnection initialized from /data/app/com.mick.we.trailerb-2/base.apk/assets/bin/Data (debug = 0)

D/Unity   ( 8333): PlayerConnection initialized network socket : 0.0.0.0 55021

D/Unity   ( 8333): PlayerConnection initialized unix socket : Unity-com.mick.we.trailerb

D/Unity   ( 8333): Multi-casting "[IP] 192.168.0.21 [Port] 55021 [Flags] 2 [Guid] 1045144479 [EditorId] 1701664046 [Version] 1048832 [Id] AndroidPlayer(motorola_XT1092@192.168.0.21) [Debug] 0" to [225.0.0.222:54997]...

D/Unity   ( 8333): InitializeScriptEngine OK (a3c60ee0)

D/Unity   ( 8333): PlayerConnection already initialized - listening to [192.168.0.21:55021]

D/Unity   ( 8333): PlayerInitEngineNoGraphics OK

D/Unity   ( 8333): GfxDevice: creating device client; threaded=1

D/Unity   ( 8333): [EGL] Request: ES 2.0 RGBA32 8888 0/0

D/Unity   ( 8333): [EGL] Found: ID[5] ES 2.0 RGBA32 8888 0/0 AFBT

D/Unity   ( 8333): ANativeWindow: (1920/1080) RequestedResolution: (0/0) EGLSurface: (1920/1080)

D/Unity   ( 8333): Renderer: Adreno (TM) 330

D/Unity   ( 8333): Vendor:   Qualcomm

D/Unity   ( 8333): Version:  OpenGL ES 3.0 V@84.0 AU@05.00.02.042.016 (CL@)

D/Unity   ( 8333): GLES:     3

D/Unity   ( 8333): GL_AMD_compressed_ATC_texture GL_AMD_performance_monitor GL_AMD_program_binary_Z400 GL_EXT_debug_label GL_EXT_debug_marker GL_EXT_discard_framebuffer GL_EXT_robustness GL_EXT_texture_format_BGRA8888 GL_EXT_texture_type_2_10_10_10_REV GL_NV_fence GL_OES_compressed_ETC1_RGB8_texture GL_OES_depth_texture GL_OES_depth24 GL_OES_EGL_image GL_OES_EGL_sync GL_OES_EGL_image_external GL_OES_element_index_uint GL_OES_fbo_render_mipmap GL_OES_fragment_precision_high GL_OES_get_program_binary GL_OES_packed_depth_stencil GL_OES_depth_texture_cube_map GL_OES_rgb8_rgba8 GL_OES_standard_derivatives GL_OES_texture_3D GL_OES_texture_float GL_OES_texture_half_float GL_OES_texture_half_float_linear GL_OES_texture_npot GL_OES_vertex_half_float GL_OES_vertex_type_10_10_10_2 GL_OES_vertex_array_object GL_QCOM_alpha_test GL_QCOM_binning_control GL_QCOM_driver_control GL_QCOM_perfmon_global_mode GL_QCOM_extended_get GL_QCOM_extended_get2 GL_QCOM_tiled_rendering GL_QCOM_writeonly_rendering GL_EXT_sRGB GL_EXT_sRGB_write_control GL_EXT_

D/Unity   ( 8333): texture_sRGB_decode GL_EXT_texture_filter_anisotropic GL_EXT_multisampled_render_to_texture GL_EXT_color_buffer_float GL_EXT_color_buffer_half_float GL_EXT_disjoint_timer_query 

D/Unity   ( 8333): ETC2 1

D/Unity   ( 8333): OPENGL LOG: Creating OpenGL ES 2.0 graphics device

D/Unity   ( 8333): InitializeGfxDevice OK

D/Unity   ( 8333): Initialize engine version: 5.2.1f1 (44735ea161b3)

D/Unity   ( 8333): AudioManager: Selecting AudioTrack output...

D/Unity   ( 8333): Begin MonoManager ReloadAssembly

D/Unity   ( 8333): Platform assembly: /data/app/com.mick.we.trailerb-2/base.apk/assets/bin/Data/Managed/UnityEngine.dll (this message is harmless)

D/Unity   ( 8333): Loading /data/app/com.mick.we.trailerb-2/base.apk/assets/bin/Data/Managed/UnityEngine.dll into Unity Child Domain

D/Unity   ( 8333): Platform assembly: /data/app/com.mick.we.trailerb-2/base.apk/assets/bin/Data/Managed/Assembly-CSharp-firstpass.dll (this message is harmless)

D/Unity   ( 8333): Loading /data/app/com.mick.we.trailerb-2/base.apk/assets/bin/Data/Managed/Assembly-CSharp-firstpass.dll into Unity Child Domain

D/Unity   ( 8333): Platform assembly: /data/app/com.mick.we.trailerb-2/base.apk/assets/bin/Data/Managed/Assembly-CSharp.dll (this message is harmless)

D/Unity   ( 8333): Loading /data/app/com.mick.we.trailerb-2/base.apk/assets/bin/Data/Managed/Assembly-CSharp.dll into Unity Child Domain

D/Unity   ( 8333): Platform assembly: /data/app/com.mick.we.trailerb-2/base.apk/assets/bin/Data/Managed/Assembly-UnityScript-firstpass.dll (this message is harmless)

D/Unity   ( 8333): Loading /data/app/com.mick.we.trailerb-2/base.apk/assets/bin/Data/Managed/Assembly-UnityScript-firstpass.dll into Unity Child Domain

D/Unity   ( 8333): Platform assembly: /data/app/com.mick.we.trailerb-2/base.apk/assets/bin/Data/Managed/Assembly-UnityScript.dll (this message is harmless)

D/Unity   ( 8333): Loading /data/app/com.mick.we.trailerb-2/base.apk/assets/bin/Data/Managed/Assembly-UnityScript.dll into Unity Child Domain

D/Unity   ( 8333): Platform assembly: /data/app/com.mick.we.trailerb-2/base.apk/assets/bin/Data/Managed/UnityEngine.UI.dll (this message is harmless)

D/Unity   ( 8333): Loading /data/app/com.mick.we.trailerb-2/base.apk/assets/bin/Data/Managed/UnityEngine.UI.dll into Unity Child Domain

D/Unity   ( 8333): Platform assembly: /data/app/com.mick.we.trailerb-2/base.apk/assets/bin/Data/Managed/UnityEngine.Networking.dll (this message is harmless)

D/Unity   ( 8333): Loading /data/app/com.mick.we.trailerb-2/base.apk/assets/bin/Data/Managed/UnityEngine.Networking.dll into Unity Child Domain

D/Unity   ( 8333): Platform assembly: /data/app/com.mick.we.trailerb-2/base.apk/assets/bin/Data/Managed/System.Core.dll (this message is harmless)

D/Unity   ( 8333): - Completed reload, in  0.126 seconds

D/Unity   ( 8333): PlayerInitEngineGraphics OK

D/Unity   ( 8333): Found 23 native sensors

D/Unity   ( 8333): Sensor :        Accelerometer ( 1) ; 0.004790 / 0.01s ; MPU6515 3-axis Accelerometer / InvenSense 

D/Unity   ( 8333): Sensor :        Accelerometer ( 1) ; 0.004790 / 0.01s ; MPU6515 3-axis Accelerometer / InvenSense 

D/Unity   ( 8333): Requested framebuffer: resolution[1920x1080], rgba[8/8/8/8], depth+stencil[on], samples[1]

D/Unity   ( 8333): Created framebuffer: resolution[1920x1080], rgba[8/8/8/8], depth+stencil[24/8], samples[0] 

D/Unity   ( 8333): Platform assembly: /data/app/com.mick.we.trailerb-2/base.apk/assets/bin/Data/Managed/System.dll (this message is harmless)

I/Unity   ( 8333): Awakening level trailer-boot

I/Unity   ( 8333): UnityEngine.Debug:Internal_Log(Int32, String, Object)

I/Unity   ( 8333): UnityEngine.Debug:Log(Object)

I/Unity   ( 8333): GameSettings:Awake() (at C:\Users\TO102081\Workspaces\unity\weons-prologue\Assets\Whispering Eons\Scripts\GameSettings.cs:38)

I/Unity   ( 8333):  

I/Unity   ( 8333): (Filename: C Line: 0)

I/Unity   ( 8333): 

D/Unity   ( 8333): Sensor :        Accelerometer ( 1) ; 0.004790 / 0.01s ; MPU6515 3-axis Accelerometer / InvenSense 

I/Unity   ( 8333): Starting level trailer-boot

I/Unity   ( 8333): UnityEngine.Debug:Internal_Log(Int32, String, Object)

I/Unity   ( 8333): UnityEngine.Debug:Log(Object)

I/Unity   ( 8333): GameSettings:Start() (at C:\Users\TO102081\Workspaces\unity\weons-prologue\Assets\Whispering Eons\Scripts\GameSettings.cs:65)

I/Unity   ( 8333):  

I/Unity   ( 8333): (Filename: C Line: 0)

I/Unity   ( 8333): 

D/Unity   ( 8333): Setting up 1 worker threads for Enlighten.

D/Unity   ( 8333):   Thread -> id: b8b70078 -> priority: 1 

I/Unity   ( 8333): Creating sceneUtils object

I/Unity   ( 8333): UnityEngine.Debug:Internal_Log(Int32, String, Object)

I/Unity   ( 8333): UnityEngine.Debug:Log(Object)

I/Unity   ( 8333): SceneUtils:get_PROXY() (at C:\Users\TO102081\Workspaces\unity\weons-prologue\Assets\Whispering Eons\Scripts\Tools\SceneUtils.cs:13)

I/Unity   ( 8333): SceneBoot:sbs() (at C:\Users\TO102081\Workspaces\unity\weons-prologue\Assets\Whispering Eons\Scripts\SceneBoot.cs:47)

I/Unity   ( 8333): UnityEngine.Events.InvokableCall:Invoke(Object[])

I/Unity   ( 8333): UnityEngine.Events.InvokableCallList:Invoke(Object[])

I/Unity   ( 8333): UnityEngine.Events.UnityEventBase:Invoke(Object[])

I/Unity   ( 8333): UnityEngine.Events.UnityEvent:Invoke()

I/Unity   ( 8333): UnityEngine.UI.Button:Press()

I/Unity   ( 8333): UnityEngine.UI.Button:OnPointerClick(PointerEventData)

I/Unity   ( 8333): UnityEngine.EventSystems.ExecuteEvents:Execute(IPointerClickHandler, BaseEventData)

I/Unity   ( 8333): UnityEngine.EventSystems.ExecuteEvents:Execute(GameObject, BaseEventData, EventFunction`1)

I/Unity   ( 8333): UnityEngine.EventSystems.TouchInputModule:ProcessTouchPress(PointerEventData, Boolean, Boolean)

I/Unity   ( 8333): UnityEngine.EventSystems.TouchInputModule:ProcessTouchEvents()

I/Unity   ( 8333): UnityEngine.EventSystems.TouchInputModule:Process

D/Unity   ( 8333): Unloading 3 Unused Serialized files (Serialized files now loaded: 0)

D/Unity   ( 8333): WARNING: Shader 

D/Unity   ( 8333): Unsupported: 'Standard (Specular setup)' - Pass 'FORWARD' has no vertex shader

D/Unity   ( 8333): WARNING: Shader 

D/Unity   ( 8333): Unsupported: 'Standard (Specular setup)' - Pass 'FORWARD_DELTA' has no vertex shader

D/Unity   ( 8333): WARNING: Shader 

D/Unity   ( 8333): Unsupported: 'Standard (Specular setup)' - Pass 'SHADOWCASTER' has no vertex shader

D/Unity   ( 8333): WARNING: Shader 

D/Unity   ( 8333): Unsupported: 'Standard (Specular setup)' - All passes removed

D/Unity   ( 8333): WARNING: Shader 

D/Unity   ( 8333): Unsupported: 'Standard' - Pass 'FORWARD' has no vertex shader

D/Unity   ( 8333): WARNING: Shader 

D/Unity   ( 8333): Unsupported: 'Standard' - Pass 'FORWARD_DELTA' has no vertex shader

D/Unity   ( 8333): WARNING: Shader 

D/Unity   ( 8333): Unsupported: 'Standard' - Pass 'SHADOWCASTER' has no vertex shader

D/Unity   ( 8333): WARNING: Shader 

D/Unity   ( 8333): Unsupported: 'Standard' - All passes removed

D/Unity   ( 8333): UnloadTime: 45.428001 ms

I/Unity   ( 8333): Awakening level trailer-menu

I/Unity   ( 8333): UnityEngine.Debug:Internal_Log(Int32, String, Object)

I/Unity   ( 8333): UnityEngine.Debug:Log(Object)

I/Unity   ( 8333): GameSettings:Awake() (at C:\Users\TO102081\Workspaces\unity\weons-prologue\Assets\Whispering Eons\Scripts\GameSettings.cs:38)

I/Unity   ( 8333):  

I/Unity   ( 8333): (Filename: C Line: 0)

I/Unity   ( 8333): 

I/Unity   ( 8333): Creating Cardboard object

I/Unity   ( 8333): UnityEngine.Debug:Internal_Log(Int32, String, Object)

I/Unity   ( 8333): UnityEngine.Debug:Log(Object)

I/Unity   ( 8333): Cardboard:get_SDK() (at C:\Users\TO102081\Workspaces\unity\weons-prologue\Assets\Cardboard\Scripts\Cardboard.cs:40)

I/Unity   ( 8333): APIGateway:CardboardTapIsTrigger(Boolean) (at C:\Users\TO102081\Workspaces\unity\weons-prologue\Assets\Whispering Eons\Scripts\Tools\APIGateway.cs:75)

I/Unity   ( 8333): APIGateway:CardboardInit()

I/Unity   ( 8333): GameSettings:Awake() (at C:\Users\TO102081\Workspaces\unity\weons-prologue\Assets\Whispering Eons\Scripts\GameSettings.cs:56)

I/Unity   ( 8333):  

I/Unity   ( 8333): (Filename: C Line: 0)

I/Unity   ( 8333): 

D/Unity   ( 8333): Platform assembly: /data/app/com.mick.we.trailerb-2/base.apk/assets/bin/Data/Managed/Boo.Lang.dll (this message is harmless)

D/Unity   ( 8333): System memory in use before: 47.3 MB.

D/Unity   ( 8333): System memory in use after: 47.3 MB.

D/Unity   ( 8333): 

D/Unity   ( 8333): Unloading 16 unused Assets to reduce memory usage. Loaded Objects now: 1302.

D/Unity   ( 8333): Total: 6.679000 ms (FindLiveObjects: 0.425000 ms CreateObjectMapping: 0.189000 ms MarkObjects: 3.079000 ms  DeleteObjects: 2.976000 ms)

D/Unity   ( 8333): 

I/Unity   ( 8333): Starting level trailer-menu

I/Unity   ( 8333): UnityEngine.Debug:Internal_Log(Int32, String, Object)

I/Unity   ( 8333): UnityEngine.Debug:Log(Object)

I/Unity   ( 8333): GameSettings:Start() (at C:\Users\TO102081\Workspaces\unity\weons-prologue\Assets\Whispering Eons\Scripts\GameSettings.cs:65)

I/Unity   ( 8333):  

I/Unity   ( 8333): (Filename: C Line: 0)

I/Unity   ( 8333): 

I/Unity   ( 8333): Creating new default cardboard screen texture 2650x1610.

I/Unity   ( 8333): UnityEngine.Debug:Internal_Log(Int32, String, Object)

I/Unity   ( 8333): UnityEngine.Debug:Log(Object)

I/Unity   ( 8333): BaseVRDevice:CreateStereoScreen() (at C:\Users\TO102081\Workspaces\unity\weons-prologue\Assets\Cardboard\Scripts\VRDevices\BaseVRDevice.cs:104)

I/Unity   ( 8333): Cardboard:get_StereoScreen() (at C:\Users\TO102081\Workspaces\unity\weons-prologue\Assets\Cardboard\Scripts\Cardboard.cs:323)

I/Unity   ( 8333): CardboardEye:UpdateStereoValues() (at C:\Users\TO102081\Workspaces\unity\weons-prologue\Assets\Cardboard\Scripts\CardboardEye.cs:173)

I/Unity   ( 8333): CardboardEye:Start() (at C:\Users\TO102081\Workspaces\unity\weons-prologue\Assets\Cardboard\Scripts\CardboardEye.cs:96)

```

**Comment by smdol** [comment]
Yes, in the new SDK, we finally correct a problem that has been there for some time, which is that if the RenderTexture has the exact same size as the screen, then the barrel distortion will make the RenderTexture's pixels bigger than screen pixels in the parts of the screen right under the eyes.  This causes fuzziness there.  So the default now is to calculate the size of the RenderTexture so that _after_ barrel distortion, on texture pixel = one screen pixel right under the eyes.  This results in a bigger texture.  But on the other hand, there's now the Stereo Scale slider on Cardboard to give you overall control of the resolution, so you can dial it back if it is too big.

**Comment by MiCkSoftware** [comment]
OK. That's perfectly understood and desirable.

**Comment by nathanmartz** [comment]
Fixed in 0.6.0.
